conf/application.conf

```diff
@@ -171,6 +173,12 @@ Dev {
     enabled = true
   }
 
+  deleteUnusedApplicationsJob {
```

Comment:
I don't think we need this as these are the default values in the provider anyway.